internal/pkg/remote/config.go

```diff
@@ -87,6 +95,31 @@ func (a *StorageApi) ListComponentsRequest(branchId model.BranchId) *client.Requ
 		})
 }
 
+// ListConfigMetadataRequest https://keboola.docs.apiary.io/#reference/components-and-configurations/copy-configuration-rows/search-component-configurations
+func (a *StorageApi) ListConfigMetadataRequest(branchId model.BranchId) *client.Request {
+	metadata := ConfigMetadataResponse{}
+	return a.
+		NewRequest(resty.MethodGet, "branch/{branchId}/search/component-configurations").
+		SetPathParam("branchId", branchId.String()).
+		SetQueryParam("include", "filteredMetadata").
+		SetResult(metadata)
+}
+
+type (
```

Comment:
Nejsem si jistej jestli to teda nemá být někde v `model.object`, ale použije se to asi jen pro parsování odpovědi z API, tak asi tady?

Reply:
Myslim, ze je to OK. Typ `ConfigMetadataResponse` by mohol byt aj private `configMetadataResponse`.
Pojde nad nim volat metody aj mimo, ale nepojde ho mimo vytvorit, alebo predat do dalsej funkcie.

Nad typom `configMetadataResponse` by som definoval nejaku metodu, ktora mi vrati nejaku map `[configKey]configMetadata`, ... myslim, ze by to parsovanie response nemalo byt az v mapper.

Reply:
👍 super, moc jsem si s tím nevěděl rady jak to uspořádat.

Reply:
@webrouse když bude vracet `map[configKey]configMetadata`, tak převod do `map[string]string` mám udělat už tady, jo? Nebo taky rovnou v té metodě?

Reply:
neviem :) ako ti to dava vacsi zmysel